src/main/java/UseCase/playSongInteractor.java

```diff
@@ -0,0 +1,30 @@
+package UseCase;
+
+import Entities.MusicPlayer;
+import Entities.Song;
+import InputBoundary.playSongInputBoundary;
+import InputData.songInputData;
+import OutputBoundary.songOutputBoundary;
+import OutputData.songOutputData;
+
+
+public class playSongInteractor implements playSongInputBoundary {
```

Comment:
I think changing the name from playSong -> playSongInteractor is great, please change the reference to this class in playSpaceInteractor from playSong -> playSongInteractor -- I believe this is why the autograde is failing.